packages/smooth_app/android/build.gradle

```diff
@@ -16,6 +16,9 @@ allprojects {
         google()
         jcenter()
     }
+    tasks.withType(JavaCompile) {
+        options.compilerArgs << '-Xlint:-options'
```

Comment:
I'm not convinced it's a good idea to remove the warnings in general.
In that specific case, I think it would make more sense to create issues in the corresponding libraries / not to use those libraries at all / find non-problematic libraries.